### ISSUE TITLE
Some students became faculty and should not be treated as students anymore

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/bibeasy#17]
this file list Julien's students: https://github.com/neuropoly/bibeasy/blob/main/bibeasy/formatting.py

used by bibeasy to format (eg undeline students), however, some students are no more students, yet they still publish with Julien (eg: Eva, Benjamin). So we need to add a logic in bibeasy to account for a date range for the student status.

**Comment by jcohenadad** [comment]
One possibility is to add the start/end data of each student

**Comment by namgo** [comment]
For clarification, `scripts/student_asterisk_helper.py` will remove the asterisk from a student if they are removed from `bibeasy.formatting.STUDENTS` but do you want the asterisk to remain there between dates the student was a member of the lab?

**Comment by jcohenadad** [comment]
> For clarification, scripts/student_asterisk_helper.py will remove the asterisk from a student if they are removed from bibeasy.formatting.STUDENTS but do you want the asterisk to remain there between dates the student was a member of the lab?

yes exactly. And for this to work, we should _not_ remove students from `bibeasy.formatting.STUDENTS`, but rather add start/end information
